python/ray/data/_internal/dataset_logger.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import os
+
+import ray
+from ray._private.ray_constants import LOGGER_FORMAT, LOGGER_LEVEL
+
+
+class DatasetLogger:
+    """Logger for Ray Datasets which, in addition to logging to stdout,
+    also writes to a separate log file at `DatasetLogger.DEFAULT_DATASET_LOG_PATH`.
+    """
+
+    DEFAULT_DATASET_LOG_PATH = "logs/ray-data.log"
+
+    def __init__(self, log_name: str):
+        """Initialize DatasetLogger for a given `log_name`.
+
+        Args:
+            log_name: Name of logger (usually passed into `logging.getLogger(...)`)
+        """
+        # Logger used to logging to log file (in addition to the root logger,
+        # which logs to stdout as normal). We set `logger.propagate` to False
+        # to ensure the file logger only logs to the file, and not stdout, by default.
```

Comment:
This documentation seems confusing to me since the class-level comment says it writes to the file *in addition to* stdout.